lib/Conversion/TorchToLinalg/Utils.cpp

```diff
@@ -527,8 +556,8 @@ Value torch_to_linalg::convertTensorToElementType(OpBuilder &b, Location loc,
                                                   Type elementType) {
   auto dtypePromoteBody = [&](OpBuilder &builder, Location loc,
                               ValueRange payloadArgs) {
-    Value elem =
-        convertScalarToDtype(builder, loc, payloadArgs[0], elementType);
+    Value elem = mlir::torch::Torch::convertScalarToDtype(
```

Comment:
`mlir::torch::Torch` was required otherwise `convertScalarToDtype` defined in `Arith/Utils.h` (included via `SCF.h`) was being used instead. However, with the change to use `arith.select` in place of `scf.if` I no longer need to include `SCF.h`, so this is reverted.